### ISSUE TITLE
Bundle com.twelvemonkeys.imageio.bmp does not exist in the workspace or the current target platform

### ISSUE THREAD

**Issue by metal450** [portfolio-performance/portfolio#3713]
I'm trying to get Eclipse setup so I can build; I believe I've followed all the instructions at https://github.com/portfolio-performance/portfolio/blob/master/CONTRIBUTING.md, but a single error remains: `Bundle com.twelvemonkeys.imageio.bmp does not exist in the workspace or the current target platform`. I'm not at all familiar with Eclipse, but I just took a guess & tried to search for "Twelvemonkeys" under Help->Eclipse Marketplace, & installed the only thing that came up (Quickimage2), but it didn't resolve.

Is something missing in the setup instructions, or did I just miss something? I searched both Github Issues & the forum for "twelvemonkeys" but got no hits. I've reloaded the target platform.

**Comment by Nirus2000** [comment]
Hello @metal450 
Have you installed the M2E packages?
![grafik](https://github.com/portfolio-performance/portfolio/assets/45203494/a5d63f60-5964-4e73-afe3-6340bd17caff)

![grafik](https://github.com/portfolio-performance/portfolio/assets/45203494/7f6b15f6-0575-41bb-9d06-df1651cd91f2)

**Comment by metal450** [comment]
Yup, installed.

So I dumped everything & started completely over from scratch, and somehow this time that error is gone, but there are different ones: 

![2024-01-06 07 56 18](https://github.com/portfolio-performance/portfolio/assets/1877409/a49e2be6-54ea-458c-8c01-6f17eca727c9)

However, PP does seem to be able to build & run, so I guess good enough.

(Man though, I've worked with a lot of IDEs in my day, but I gotta say...never seen anything that felt even close to as clunky/unintuitive as Eclipse. 😵‍)
